tests/components/utility_meter/test_sensor.py

```diff
@@ -240,6 +240,13 @@ def gen_config(cycle, offset=None):
     )
 
 
+async def test_self_reset_quarterly(hass):
+    """Test quarterly reset of meter."""
+    await _test_self_reset(
+        hass, gen_config("quarterly"), "2017-12-31T23:59:00.000000+00:00"
```

Comment:
testing at the end of the year is poor, can you move the test to the end of the 1st quarter ?

Reply:
Done